frontend/src/components/Site.jsx

```diff
@@ -154,6 +171,9 @@ const Site = () => {
 					>
 						{siteDescription}
 					</Typography>
+					<Typography>
+						{location}, {"REGION WOULD GO HERE (todo in refactor)"}
```

Comment:
Thats funny, I don't see an issue created for this task... is github broken? I am confused.

Reply:
#2